parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2588,6 +2588,8 @@ doSlurpUpdates typeEdits termEdits deprecated b0 =
     Just split -> [ BranchUtil.makeDeleteTermName split (Referent.Ref old)
                   , BranchUtil.makeAddTermName split (Referent.Ref new) oldMd ]
       where
+      -- oldMd is the metadata linked to the old definition
+      -- we relink it to the new definition
       oldMd = BranchUtil.getTermMetadataAt split (Referent.Ref old) b0
```

Comment:
I needed to know what this line did, so I added a comment.

Reply:
lol